codec-native-quic/src/main/resources/META-INF/native-image/io.netty.incubator/netty-incubator-codec-native-quic/resource-config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "resources":{
+  "includes":[
+    {
+      "condition":{"typeReachable":"io.netty.incubator.codec.quic.Quiche"},
+      "pattern":"\\QMETA-INF/native/libnetty_quiche_osx_x86_64.jnilib\\E"
+    },
```

Comment:
```suggestion
    {
      "condition":{"typeReachable":"io.netty.incubator.codec.quic.Quiche"},
      "pattern":"\\QMETA-INF/native/libnetty_quiche_osx_x86_64.jnilib\\E"
    },
    {
      "condition":{"typeReachable":"io.netty.incubator.codec.quic.Quiche"},
      "pattern":"\\QMETA-INF/native/libnetty_quiche_osx_aarch_64.jnilib\\E"
    },
```

Reply:
Can you add a comment for this...

Reply:
ok I will add the resource so that we don't need to add it later and in addition I'll add a comment that this is still not supported.